src/assets/style.css

```diff
@@ -327,16 +327,21 @@ main h2.tip-title {
   font-weight: bold;
 }
 
-.tip img {
+.tip .tip-image {
   grid-column: 2;
   grid-row: 1 / span 3;
   object-fit: cover;
```

Comment:
`object-fit: cover` should be on the `.tip .tip-image img` rule instead. This particular property makes the image always cover the entire area that's available to it.